lighthouse-extension/app/src/lighthouse-background.js

```diff
@@ -56,6 +57,42 @@ window.runAudits = function(options, audits) {
       });
 };
 
+window.getListOfAudits = function() {
+  return _flatten(
+    configJSON.aggregations.map(aggregation => {
+      if (aggregation.items.length > 1) {
```

Comment:
Yes exactly === 1 feels better because it's done for the edge cases like Performance & best practises